### PR TITLE
move CRVResponse text files to data

### PR DESCRIPTION
**Pull request by rlcee**
The cmake Offline build did not install these files correctly.  This is a fix, and also a better separation of directory content.  Checked by running several validation exe's in both builds.

### CODE REVIEW

**Comment by kutschke** [comment]
@oksuzian and @ehrlich-uva I added you as reviewers just to give you a heads up on some bookkeeping changes that Ray is making.  I don't plan to wait for you to review before merging but I would appreciate it if you approve once you have had a look.

**Review by kutschke — APPROVED** [review]
Looks good.

**Review by ehrlich-uva — APPROVED** [review]

**Pull request merged**
